stages/org.osbuild.dnf

```diff
@@ -101,6 +102,8 @@ def main(tree, options):
     ]
 
     cmd = base_cmd + [operation] + packages
+    if exclude_packages:
+        cmd += ["--exclude"] + exclude_packages
```

Comment:
Is `exclude_packages` a string or a list? I'd say it's a list, in which case I think you'd need to do
```python
exclude_packages = options.get("exclude-packages", [])
# ...
for x in exclude_packages:
    cmd += ["--exclude", x]
```

Reply:
Ah, of course! Thanks!